lib/hydra/derivatives.rb

```diff
@@ -118,7 +118,7 @@ def constantize_processor(processor)
     end
 
     def transform_datastream(file_name, transform_directives, opts={})
```

Comment:
I just created this rspec to test the deprecated method
transform_datastream.  It's the same as transcoding_spec.rb, except it uses
the deprecated method transform_datastream.  Take a look:

https://gist.github.com/blancoj/2550a7b0c40dbc11c585

Along with the previous change I put in, which can be seen at the bottom of
this link:

https://github.com/projecthydra/hydra-derivatives/pull/69/files#r36449182

I then ran the rspec against this new file and I got the same errors I got
when testing spec/units/transcoding_spec.rb.  Since I don't have all the FITS
and FFMpeg and kdu_compress on my local machine, I expect errors. I'm
confident that the full rspec will pass once I submit these changes.

Before I submit this PR, I just want to make sure this makes sense.

Thank you!
Jose


On Fri, Aug 7, 2015 at 1:10 PM, stkenny <notifications@github.com> wrote:

> In lib/hydra/derivatives.rb
> <https://github.com/projecthydra/hydra-derivatives/pull/69#discussion_r36539210>
> :
>
> > @@ -118,7 +118,7 @@ def constantize_processor(processor)
> >      end
> >
> >      def transform_datastream(file_name, transform_directives, opts={})
>
> Well if the change in this pull request is applied then the behaviour will
> be as before. Passing an empty options hash isn't a problem as long as you
> transforming an image datastream.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/projecthydra/hydra-derivatives/pull/69/files#r36539210>
> .
>